### ISSUE TITLE
[Accessibility]A11y_VSCode_Editor_ScreenReader: Jaws does not read any information about "suggestions" that comes after when we type any word in the editor.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52433]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
Jaws Version: 18.0.5038

**Repro Steps:**
1. Launch VS Code while Jaws is ON.
2. Open any file in editor and type some word.

**Actual:**
Jaws does not read any information about "suggestions" that comes after when we type any word in the editor.

**Expected:**
Jaws should announce some information when any suggestion appears after typing any word.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know about all the words,functions or datatype that displays while typing the word. 

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Editor Jaws does not read suggestions and focus order.pptx](https://github.com/Microsoft/vscode/files/2118800/A11y.VSCode.Editor.Jaws.does.not.read.suggestions.and.focus.order.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.
